src/main/java/org/jabref/logic/bibtex/BibDatabaseMerger.java

```diff
@@ -0,0 +1,65 @@
+package org.jabref.logic.bibtex;
```

Comment:
Since this is not related to writing bibtex (as the rest of the package), I suggest to move it together with the duplication check to a new `logic.database` package in parallel to the existing `model.database`. @JabRef/developers opinions?

Reply:
I moved it to the recommended package :).